app/views/trips/_trip.html.erb

```diff
@@ -0,0 +1,28 @@
+<div class="container-lg" style="border-style: solid; border-width: 1px; border-radius: 7px; padding: 10px 25px; margin-top: 10px">
+    <a href=<%= "/trips/" + trip.id.to_s %> style="text-decoration: none; color: black">
+      <div class="row">
+        <div class="col">
+          <div class="row">
+            <h4>
+              <%= trip.name %>
+            </h4>
+          </div>
+          <div class="row">
+            <%= trip.start_date.strftime("%b %e") + " - " + trip.end_date.strftime("%b %e, %Y")%>
+          </div>
+          <div class="row">
+            <%= "in " + trip.location %>
+          </div>
+        </div>
+        <div class="col-8">
+          <div class="row">
+            <% trip.own_photos.each do |photo| %>
+              <div class="col">
+                <img src=<%= photo.url %> class="img-fluid"  style="max-height: 300px; border-radius: 5px">
```

Comment:
There's also an [`image_tag`](https://apidock.com/rails/ActionView/Helpers/AssetTagHelper/image_tag) method you can use instead of the `<img>` element.